_data/sites.json

```
@@ -253,6 +253,18 @@
         ]
     },
 
+    {
+        "name": "Academic Torrents",
+        "url": "https://academictorrents.com",
+        "difficulty": "hard",
+        "notes": "You must send an email with your account details to contact@academictorrents.com",
```

```suggestion
        "notes": "You must send an email with your account details.",
```

Since the e-mail is on a separate field already, no need for us to mention it here.